pkgs/applications/blockchains/oxen/default.nix

```
@@ -0,0 +1,67 @@
+{ stdenv, lib, fetchurl, fetchFromGitHub, fetchpatch
+, cmake, pkg-config
+, boost, openssl, unbound
+, pcsclite, readline, libsodium, hidapi
+, rapidjson
+, curl, sqlite
+, trezorSupport ? true
+,   libusb1  ? null
+,   protobuf ? null
+,   python3  ? null
```

```suggestion
, trezorSupport ? true
, libusb1
, protobuf
, python3
```
They cannot be null. If they are null there is a mistake here.